front_end/helper.py

```diff
@@ -243,3 +243,19 @@ def show_hidden(request_handler):
     if "show_hidden" not in request_handler.request.query_arguments:
         return False
     return request_handler.request.query_arguments["show_hidden"][0].decode().lower() == "true"
+
+def get_days_months():
```

Comment:
Please make this function name more descriptive. The function name is very generic, but it seems that it is accomplishing a specific purpose. Also consider whether content.py is the best place for this function.  If this function is specific to log summarization, it should go in that script. But if it can be used more generally, it can go here.